ApiContract/response.json5

```diff
@@ -0,0 +1,45 @@
+//Response data
+{
+	"advertisement" : {
+		//[Mandatory] id of the advertisement
+		"id" : "P123",
+		//[Mandatory] title of the advertisement
+		"name" : "Watch" ,
+		//[Mandatory] description of the advertisement
+		"description" : "something",
+		//[Mandatory] images of the product in the advertisement (Maximum files - 5)
+		"heroImageUrl" : "www.example.com/imageName",
+		//[Mandatory] 
+		"price" : 
+		{
+			// price of the product in the advertisement (in Rupees)
+			"amount" : "300",
+			// Boolean flag specifying whether price is negotiable or not
+			"isPriceNegotiable" : "true"
+			
+		},
+		//[Mandatory] category of the product in the advertisement
+		"category" : "Accessories",
+
+		//[Optional] images of the product
+		"imageUrls" : [
+            {
+                "url" : "www.example.com/imageName",
```

Comment:
again a contract should be self describing. so by "www.example.com/imageName" value i mean "www.example.com/watchFrontImage"